### DISCUSSION TITLE
Nexus 9P Slim PWM fan hub

### DISCUSSION THREAD

**Discussion by aLLaN8602** [Rem0o/FanControl.Releases#842] [Q&A]
Hello.

I tryed this software some weeks ago, but i had an issue immediately but forgot to write her about my problem. I have Fractal Torrent (RGB ver.) case that has this "Nexus 9P Slim PWM fan hub" installed from factory. I have 2 front fans (using splitter connector for connecting those 2 fans into into single fan port), 3 bottom, 1 rear, 1 CPU and 2 on GPU. I'm using bottom 3 fans through this case fan hub, but the software can't get info or change rpm through it somehow. Since ASUS Strix B550-E board doesn't have enough fan connectors on board (5 if i remember correctly), i can't connect all my fans directly and have no way of using this software, sadly. ASUS own software somehow can read info and change rpm on those fans. i hope you can fir the issue soon, so i can test your software again someday.

Thanks in advance,
Allan

**Comment by torifat** [comment]
https://support.fractal-design.com/support/solutions/articles/4000180230-torrent-compact

> The main header is marked in the manual, and on the hub, it's marked as FAN1. This needs to have at least 1 PWM fan connected to it in order for you to control the fans on the hub, considering it's the only header that receives and sends signals to the motherboard. It does not need to be the CPU-cooler connected to the header, but it has to be a 4-pin PWM unit. You don't have to connect the hub to the CPU_FAN header either, but have it on a 4-pin PWM-header (avoid the AIO_PUMP header, since it's sometimes set to run at 100%).

For people facing the same issue- you need to **connect a PWM fan to the white FAN1 header** for it to work.

**Reply by aLLaN8602** [reply]
But i already have it setup like that. I knew about the fan hubs in general that only the first header usually sends and recieves signal and other headers work accordingly, otherwise i wouldnt be able to use it through ASUS own software either.

**Reply by torifat** [reply]
Hmm, might be something related to Asus board then.

I shared the info mostly as PSA for people who might face similar problem 🙂. I assumed you solved your issue already since the post is almost 7 months 😅.

**Reply by aLLaN8602** [reply]
I still have the issue. Using ASUS own software, otherwise i couldn't control bottom 3 fans powering through the fan hub.
